src/components/dialogs/line-attach-to-voltage-level-dialog.js

```diff
@@ -162,6 +163,7 @@ const LineAttachToVoltageLevelDialog = ({
                     : '',
         });
     const voltageLevelOrIdRef = useRef();
+    voltageLevelOrIdRef.current = voltageLevelOrId;
```

Comment:
Could use
`const voltageLevelOrIdRef = useRef(voltageLevelOrId);`
instead of
```
const voltageLevelOrIdRef = useRef();
voltageLevelOrIdRef.current = voltageLevelOrId;
```